plugins/catalog/src/components/CatalogKindHeader/CatalogKindHeader.tsx

```diff
@@ -25,12 +24,13 @@ import {
   Theme,
 } from '@material-ui/core';
 import {
-  catalogApiRef,
   EntityKindFilter,
   useEntityList,
 } from '@backstage/plugin-catalog-react';
-import useAsync from 'react-use/lib/useAsync';
-import { useApi } from '@backstage/core-plugin-api';
+import {
+  filterKinds,
+  useAllKinds,
+} from '@backstage/plugin-catalog-react/src/utils';
```

Comment:
note that the `/src/utils` bit here is what's causing the compilation to fail

Reply:
Yes. Sorry about that. Still need your input here before I can fix it: https://github.com/backstage/backstage/pull/14366#discussion_r1036055989